docs/adoption.md

```diff
@@ -141,7 +141,13 @@ To deploy the Housing Management System API, you must populate github actions wi
 | `RESOURCE_GROUP_NAME`                  | The resource group name housing your Terraform state file                                                         |
 | `STORAGE_ACCOUNT_NAME`                 | The name of the Azure Storage Account to house your Terraform state file                                          |
 | `STATE_CONTAINER_NAME`                 | The name of the Azure Blob Storage container to house your Terraform state file                                   |
-| `STATE_KEY_NAME`                       | The file path and name of your Terraform state file                                                               |
+| `STATE_KEY_NAME`                       | The file path and name of your Terraform state file                                                             
+| `COSMOS_DATABASE_ID`        | DocumentDB (e.g. CosmosDB) database name , e.g. `housing-repairs` |
```

Comment:
Given the secret name contains the specific database (CosmosDB) there isn't a need to have a generic comment.